src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        return new Ball(new ColorSupplier().getRandomColor(), random.nextInt(100));
```

Comment:
All magic numbers in your code should be constants. The number 100 should be extracted into a constant with a meaningful name to avoid magic numbers and to clarify what it represents.